### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'outputs'

### ISSUE THREAD

**Issue by bugsuse** [blendernc/blendernc#113]
Blender raised AttributeError when using blendernc load Gebco file on mac according to the [tutorial](https://blendernc.readthedocs.io/en/latest/examples/import_gebco_netCDF.html)

```python
Python: Traceback (most recent call last):
  File "/Users/yanglee/Library/Application Support/Blender/2.92/scripts/addons/blendernc/operators.py", line 334, in execute
    texcoord_link = texcoord.outputs.get("UV")
AttributeError: 'NoneType' object has no attribute 'outputs'

location: <unknown location>:-1
```
<img width="1437" alt="截屏2021-07-10 下午10 02 35" src="https://user-images.githubusercontent.com/16102453/125165747-ecf42a00-e1ca-11eb-90eb-8da9757f364b.png">

I tried to update blendernc but not works. Could you help me to solve it? Thanks a lot!

**Comment by josuemtzmo** [comment]
Hello @bugsuse, I haven't been able to reproduce this issue, I tried Blender 2.92 and BlenderNC `master` branch, are these the versions you are using? 

I can only notice one small detail, but that shouldn't be generating the `AttributeError` message, due to the high resolution of the GEBCO dataset (86400 x 43200 pixels), blenderNC currently supports images datasets with resolution   of 5120 × 2880 pixels. This can be achieved by changing the resolution percentage to ~92%. 

Anyhow, in order for me to have a better idea of what is causing the main `AttributeError` report, could you please attach an image of the `ShaderEditor` titled `BlenderNC_default` and the `BlenderNC` node tree editor titled `BlenderNC`. See image below:
 
<img width="1920" alt="Screen Shot 2021-07-11 at 12 41 08 am" src="https://user-images.githubusercontent.com/9144063/125166809-b1fcf100-e1e0-11eb-83a6-bf3c4d93b0e0.png">

**Comment by bugsuse** [comment]
Hey, @josuemtzmo I use Blender 2.93 and blendernc 0.3.2. I tested it in the morning and it works now. I just set the language from Chinese to English and resolution 90%. It also works that even though change language back.  Last night it did not work even if the resolution was set to 90%. In a word, it's so strange! Maybe I can close the comment now?

**Comment by bugsuse** [comment]
It is possible that the language setting is causing this problem. Blender raised AttributeError when applying material after rendered it if setting language to Chinese, but normal in English.

**Comment by josuemtzmo** [comment]
Please leave this issue open, I will investigate if the translation of Blender is creating this issue. In the meantime, I would recommend you to use the addon in English. Apologize for the language restriction at the moment, but I will look into fixing it.

**Comment by josuemtzmo** [comment]
In fact, this was an issue generated by the Blender translation API, translating the keywords: "Output", "Input", and "Resolution". Full support for languages will be implemented in a future release, however, in the meantime #102 adds partial support for those particular words.

**Comment by bugsuse** [comment]
Great!

**Comment by josuemtzmo** [comment]
I'm closing this issue, as now it has been solved and implemented into version 0.4.0
